File: java/test/com/thoughtworks/cep/conference/FeatureTogglesTest.java
package com.thoughtworks.cep.conference;
import org.junit.jupiter.api.Test;
import java.util.Arrays;

import static org.junit.jupiter.api.Assertions.*;

public class FeatureTogglesTest {

    @Test
    public void returnsTrueWhenFeatureIsEnabled() {
        String enabledFeature = "feature1";

        FeatureToggles toggleService = new FeatureToggles(Arrays.asList(enabledFeature));

        assertTrue(toggleService.isEnabled(enabledFeature));
    }

    @Test
    public void returnsFalseWhenFeatureIsNotEnabled() {
        FeatureToggles toggleService = new FeatureToggles(Arrays.asList());

        assertFalse(toggleService.isEnabled("feature1"));
    }
}

File: kotlin/test/com/thoughtworks/cep/conference/FeatureTogglesTest.kt
package com.thoughtworks.cep.conference

import kotlin.test.Test
import kotlin.test.assertFalse
import kotlin.test.assertTrue

class FeatureTogglesTest {
    @Test
    fun returnsTrueWhenFeatureIsEnabled() {
        val enabledFeature = "feature1"
        val toggleService = FeatureToggles(listOf(enabledFeature))
        assertTrue(toggleService.isEnabled(enabledFeature))
    }

    @Test
    fun returnsFalseWhenFeatureIsNotEnabled() {
        val toggleService = FeatureToggles()
        assertFalse(toggleService.isEnabled("feature1"))
    }
}